lib/apollo/models/sanitizer.rb

```diff
@@ -98,12 +98,19 @@ def blacklist(type:nil, key:nil)
       self
     end
 
-    def type_mapping(key:nil, value:nil)
+    def type_mapping(key: nil, value: nil)
       self.set([:type_mappings, key], value)
 
       self
     end
 
+    def update_default_ttl(type, value)
+      updatedTTL = { type => { "default_ttl" => value } }
+      self.set(:ttl_rules, updatedTTL)
```

Comment:
do we need to take care of case like this `sanitizer.update_default_ttl("useractions", "366d").update_default_ttl("events", "1d").update` ?